readme.md

```diff
@@ -450,3 +461,36 @@ body, .foo {
   [RULE_END]  
 ]
 ```
+
+#### JavaScript Function Value
```

Comment:
And the problem is that not just styled-components will be slow and complex because of that, but every parser that wants to support all of ISTF. Essentially bad design decisions would spread out.